src/main/java/org/dishorg/dishorg/Recipe.java

```diff
@@ -1,24 +1,29 @@
 package org.dishorg.dishorg;
 
-import lombok.Data;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.Getter;
+import lombok.Setter;
 
-import javax.persistence.Column;
-import javax.persistence.ElementCollection;
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.Id;
+import javax.persistence.*;
 import java.util.List;
+import java.util.Set;
 
-@Data
+@Getter
+@Setter
 @Entity
 public class Recipe {
+    @JsonIgnore
+    @ManyToMany
+    @JoinTable(
+            name = "grocery_list_recipe",
+            joinColumns = @JoinColumn(name = "recipe_id"),
+            inverseJoinColumns = @JoinColumn(name = "grocery_list_id"))
+    Set<GroceryList> groceryLists;
     private @Id
     @GeneratedValue
     Long id;
-
```

Comment:
Put a blank line between data members to make things more readable. Same for other data members in this class.